apps/dialtone-documentation/docs/about/whats-new/posts/2024-7-5.md

```diff
@@ -0,0 +1,58 @@
+---
+heading: New search in the Tokens Page and new Empty State Component
+author: Nina Repetto
+posted: '2024-7-5'
+---
+
+<BlogPost :author="$frontmatter.author" :posted="parse($frontmatter.posted, 'y-M-d', new Date())" :heading="$frontmatter.heading">
+
+**👋 Hey there,** we're excited to share some updates with you:
+
+* We redesigned our [Tokens Page](/tokens) and now you can easily search and copy any token.
+
+![New Tokens Search](/assets/images/token-search.gif)
+
+* We are rolling out a new component, the [Empty State Component](/components/empty-state.html).
+
+![Empty State example](/assets/images/empty-state.png)
```

Comment:
We can summarize the updates here via text, but the images should be moved to their respective sections.

Reply:
I just wanted to show an image in the first 10 lines, to make sure people see the email. Maybe I can move the token gif to the start of the tokens section and at least show that one.

Reply:
I ended up removing the image for the empty state, because it looked like something was really missing